cmd/keymasterd/app.go

```diff
@@ -209,6 +209,7 @@ type RuntimeState struct {
 	db                           *sql.DB
 	dbType                       string
 	cacheDB                      *sql.DB
+	dbDone                       chan struct{}
```

Comment:
Consider changing the type to `chan <- struct{}` so that you can enforce only one place is able to read from the chanel.